awxkit/awxkit/cli/options.py

```diff
@@ -216,6 +225,9 @@ def jsonstr(v):
                 if param['type'] == 'id' and not kwargs.get('help'):
                     kwargs['help'] = 'the ID of the associated  {}'.format(k)
 
+                if param['type'] == 'list_of_ids':
+                    kwargs['help'] = 'a list of comma-delimited {} to associate (IDs or unique names)'.format(k)
```

Comment:
> or unique names

Really? I thought that neither the API or CLI did this.

Reply:
Yep.

Reply:
so shouldn't this just be IDs?

Reply:
I've specified a custom type which allows IDs *or* name lookups, just like other primary-key based flags:

https://github.com/ansible/awx/pull/5156/files#diff-79a0e143704892fa893998008778a07fR23

Reply:
oh, the CLI is doing it, in pk_or_name_list